functions/Save-DbaCommunitySoftware.ps1

```diff
@@ -15,6 +15,7 @@ function Save-DbaCommunitySoftware {
         * DarlingData: https://github.com/erikdarlingdata/DarlingData
         * SQLWATCH: https://github.com/marcingminski/sqlwatch/releases
         * WhoIsActive: https://github.com/amachanic/sp_whoisactive/releases
+        * DbaMultitool: https://github.com/LowlyDBA/dba-multitool/releases
```

Comment:
Have you tested downloading a release and using that file for Save-DbaCommunitySoftware with -LocalFile? Because the zip structure is different and might not work. See code for the FirstResponderKit, that is also downloadable as branch or as release.

Reply:
Good catch! Updated to handle that.